index.js

```diff
@@ -82,21 +127,40 @@ const checkSourceIsFile = (stat, source) => {
 	}
 };
 
+/**
+ * @param {string} source
+ * @param {string} destination
+ * @param {import('.').Options} options
+ */
 module.exports.sync = (source, destination, options) => {
 	if (!source || !destination) {
 		throw new CpFileError('`source` and `destination` required');
 	}
 
 	options = {
-		overwrite: true,
+		...defaultOptions,
 		...options
 	};
 
 	const stat = fs.statSync(source);
 	checkSourceIsFile(stat, source);
 	fs.makeDirSync(path.dirname(destination));
 
-	const flags = options.overwrite ? null : fsConstants.COPYFILE_EXCL;
+	let flags = 0;
+	if (!options.overwrite) {
+		flags |= fsConstants.COPYFILE_EXCL;
+	}
+
+	if (options.clone === true) {
+		flags |= fsConstants.COPYFILE_FICLONE;
+	} else if (options.clone === 'force') {
+		if (!Object.prototype.hasOwnProperty.call(fsConstants, 'COPYFILE_FICLONE_FORCE')) {
+			throw new CpFileError(`Node ${version} does not understand cloneFile`);
```

Comment:
The flags were added in Node 10 and this package targets that version, so I think this check is moot.

https://github.com/nodejs/node/commit/a16d88d9e9a6581e463082549823189aba25ca76